### PR TITLE
Fix UI amount keyboard, outline button

### LINKED ISSUE #3108: [FEATURE]  Keyboard Should be in less space
### Please confirm the following:

- [X] I've checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.
- [X] I've requested a single (only one) feature/change in this issue. It complies with the One Request Per GitHub Issue (ORPGI) rule.
- [X] My issue is well-defined and describes how it should be implemented from UI/UX perspective.

### What do you want to be added or improved?

As a user I want to keyboard of the app should be in less spaces to access the number easily

### Why do you need it?

- To access the numbers of the keyboard more easily with one hand also

### How do you imagine it?

Like we can reduce the space between the rows

### PR DESCRIPTION
**Pull request by isaacnguyen0809**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?
+ Adjust padding for responsive UI

## Screen shot
| Before    | After |
| -------- | ------- |
|Screenshot_20240410_085350  | Screenshot_20240410_085553 |
|small_account | fixed_small  |
|a |large_amount|
|c |small_cal|
|b|large_cal|
||small_amount|

- Closes #3108

### CODE REVIEW

File: temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt [hunk 1]
```diff
@@ -76,10 +76,10 @@ fun BoxWithConstraintsScope.AmountModal(
         mutableStateOf(
             if (currency.isNotEmpty()) {
                 initialAmount?.takeIf { it != 0.0 }?.format(currency)
-                    ?: ""
+                    ?: "0"
             } else {
                 initialAmount?.takeIf { it != 0.0 }?.format(decimalCountMax)
-                    ?: ""
+                    ?: "0"
```
Comment (ILIYANGERMANOV) [inline temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt:82]: That's not desired, let's keep it empty so it works and the user sees the hint. Why do we want "0"?

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
The UI improvements looks good 💯 Lets remove the `"0"` thing, it's problematic

File: temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt (hunk 1 above)
Comment (isaacnguyen0809) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt:82]: oh my bad, i already remove it in previous PR. Maybe when I reverted to capture the old UI, I accidentally added it again

Comment (ILIYANGERMANOV) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt]: @isaacnguyen0809 can you retest that the Amount modal and calculator works well and looks good in:
- empty input-> open calc -> type
- enter amount -> open calc -> type
- cases like this

A screen recording will be appreciated and then we'll merge.

Comment (isaacnguyen0809) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt]: here is the result :
https://github.com/Ivy-Apps/ivy-wallet/assets/57831153/14587150-7707-49c7-935d-d283b1b91c5c

Comment (ILIYANGERMANOV) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt]: Merging 💯 Just to check quickly it work when you click "Set" right?

Comment (isaacnguyen0809) [reply temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt]: https://github.com/Ivy-Apps/ivy-wallet/assets/57831153/25260ae0-6c24-41ee-8dc0-e1c0f9046245

yes, it worked perfectly

**Pull request merged**